### ISSUE TITLE
Update to support current Ember.JS Modules API

### ISSUE THREAD

**Issue by OakBehringer** [nypublicradio/ember-poll#4]
```DEPRECATION: Importing from the `ember-service` module has been deprecated. Please use the new module imports:

import Service from '@ember/service'
```

ETC - 

https://www.emberjs.com/blog/2017/10/11/ember-2-16-released.html

**Comment by noslouch** [comment]
fixed in `v1.4.0`

**Comment by OakBehringer** [comment]
100

Thanks!
